lv.yml

```diff
@@ -153,6 +161,18 @@ lv:
       service_attendance: pasākuma veids
       user: lietotājs
       web_hook: webhook
+    date:
```

Comment:
Please remove date section as it includes system specific input. If you need any changes from the default English date/time then please let me know where it is in the app and what change you want then I can modify it.